unity-renderer/Assets/Scripts/MainScripts/DCL/Helpers/ProfanityFiltering/ProfanityFilterSharedInstances.cs

```diff
@@ -0,0 +1,13 @@
+﻿public static class ProfanityFilterSharedInstances
+{
+    private static RegexProfanityFilter regexFilterInstance;
+
+    public static RegexProfanityFilter regexFilter
```

Comment:
Maybe we should give this instance a more specific name. In case we have multiple ways of filtering by using a regex.

Reply:
I dont recall a better name for it now. Any suggestion? Maybe change it if a modification really comes up?